data/locale.yml

```diff
@@ -30,10 +30,15 @@ keyboard_modal:
 docs:
   title: Electron Documentation
   description: See all of the <a href="/docs/all">docs on one page</a> or check out the <a href="/docs/faq">FAQ</a>.
-  guides: Guides
+  guides: "Guides: Getting Started with Electron"
   api_reference: API Reference
   api_structures: API Structures
   advanced: Advanced
+  detailed: Detailed Guides
```

Comment:
I know it's already been changed but this title breaks over two lines (as the only one). Can we do "In Detail" instead?

![Selection_004](https://user-images.githubusercontent.com/41970/58785941-dbe98f80-85e6-11e9-88b3-b55f4769f18e.png)

Reply:
If we do that (which I'm fine with), we should get rid of that phrase from the subheaders.

Reply:
At the risk of bikeshedding -- 'Guides' seems more expressive to me than 'In Detail'

Reply:
Having "Guides: Get Started with Electron" followed by "Guides" without a modifier seems a little odd to me.

Reply:
You're right, it does. Maybe "Detail" would be better here after all.

TBH though I think this is a symptom of a larger problem, that there is a lot of arbitrary organization here. For example, "Debugging the main process" is under "Getting Started", but "Debugging on Windows" is under Advanced".

That's out-of-scope for this PR, but IMO that kind of organization is something we should do in a followup

Reply:
True, I agree strongly! I want to tackle that next, but I'll need to do it over in `electron/electron`.

Reply:
Let's go with `In Detail` as the last change and make additional changes incrementally, otherwise I fear that we'll end up bikeshedding us to death 😅